lib/msal-common/src/utils/UrlUtils.ts

```diff
@@ -64,11 +65,23 @@ export function getDeserializedResponse(
 /**
  * Utility to create a URL from the params map
  */
-export function mapToQueryString(parameters: Map<string, string>): string {
+export function mapToQueryString(
+    parameters: Map<string, string>,
+    encodeExtraParams: boolean = true,
```

Comment:
You don't need to declare both `encodeExtraParams` and `extraQueryParameters` params here. One `skipEncondingParams` param should be enough.

Reply:
The reason I have both params is because we want to encode all the other params that aren't extra (for example, we don't want the scopes to have spaces in between them). The boolean is used to tell whether to skip and the StringDict tells us which params to skip if we need to. Unless there's another way to determine which scopes are extra I think we need to keep both

Reply:
Gotcha. Thanks for the details.